assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -46,25 +46,29 @@ RUN pip install git+https://github.com/stanford-futuredata/megablocks.git@5897cd
 # COPY . .
 # RUN pip install -e ./ --no-cache-dir
 
+# Flash attention needed for phi-3-mini-v
+RUN pip install packaging
+RUN pip install flash-attn --no-build-isolation
+
 # When copied to assets repo, change to install from public pypi
-RUN pip install llm-optimized-inference==0.1.7 --no-cache-dir
+RUN pip install llm-optimized-inference==0.1.8 --no-cache-dir
 
 RUN pip uninstall -y vllm
 
-COPY ./vllm.patch ./vllm.patch
-
+# Apply patch
+COPY ./vllm_phiv.patch ./vllm_phiv.patch
 RUN git clone https://github.com/vllm-project/vllm.git && \
     cd vllm && \
-    git checkout 0ce0539d4750f9ebcd9b19d7085ca3b934b9ec67 && \
-    git apply ../vllm.patch && \
+    git checkout f8e7adda21810104382bdf3febe3ea02c72f7348 && \
+    git apply ../vllm_phiv.patch && \
```

Comment:
You can also delete the vllm.patch file since it is not going to be used and shouldn't be needed in the future